services/web/server/src/simcore_service_webserver/projects/_crud_api_create.py

```diff
@@ -133,7 +133,7 @@ async def _copy_project_nodes_from_source_project(
     db: ProjectDBAPI = ProjectDBAPI.get_from_app_context(app)
 
     def _mapped_node_id(node: ProjectNode) -> NodeID:
-        return NodeID(nodes_map[NodeIDStr(f"{node.node_id}")])
+        return NodeID(nodes_map[TypeAdapter(NodeIDStr).validate_python(f"{node.node_id}")])
```

Comment:
here I wonder why we do the validation. the NodeIDStr is an annotated string. we pass a string via the f-string... is that not too much?
actually Pylance is happy if you just remove NodeIDStr from here.